docs/CHANGELOG.MD

```diff
@@ -4,7 +4,8 @@
 
 ### Fixes
 
-* [vmuser](https://docs.victoriametrics.com/operator/api.html#vmuser): [Enterprise] fixes ip_filters indent for url_prefix. Previously it wasn't possible to use ip_filters with multiple target refs
+- [vmoperator](https://docs.victoriametrics.com/operator/) follow up after [PR #742](https://github.com/VictoriaMetrics/operator/pull/742) - fix error with logger flags.
```

Comment:
if it wasn't released yet, then it is not actually a fix.

Reply:
Indeed. Fixed